docs/apps/rbac/for_users.md

```diff
@@ -0,0 +1,126 @@
+# Using DAB Role-Based Access Control
+
+Access control allows or denies requests from a user based on some permission specification.
+Roles are an implementation detail of some access control systems, such as this one.
+
+## Model Permissions
+
+Permissions are defined on a per-model basis, like **permission to change inventory**,
+which represents the permission to modify an inventory, but _which_ inventory
+is still unclear when looking at the permission itself.
+
+Permissions combine two pieces of information
+
+ - The **action** being taken
+ - The **content type** the action applies to
+
+## Role Definitions
+
+The DAB RBAC system may use the word "role" or "role definition" interchangeably.
+The "role definition" wording is help clarify that it is abstract,
+meaning that it does not have an object association.
+
+Roles contain two functional pieces of information (excluding things like its `name`)
+
+ - A list of **permissions** the role gives to all asignees
+ - A **content type** restriction
+
+## Role User Assignments
+
+When you give a user some permissions, 3 pieces of information are involved
+
+ - The **role**
+ - The **user** getting the permissions listed in that role
+ - The **object** this assignment is applied to
+
+This is illustrated in the image below. 1 user gets 1 or more permissions to the object.
+
+![User getting permission to an object](images/user_obj.svg)
+
+## Role Team Assignments
+
+Teams are treated like other objects in the system by DAB RBAC,
+with the exception that its **member** permission acts as a multiplier.
+By receiving a team's member permission, you automatically receive all
+permissions that the team has.
```

Comment:
One more TODO I'd like to illustrate: A new diagram showing team membership _as a role assignment_ via the team member role or admin role, thus showing how that role assignment defines the grouping that the subsequent diagram shows. The point is to quickly cover this to avoid confusion and say "when we draw a line around users, this is what we mean".